plugin.rb

```diff
@@ -85,6 +85,18 @@ def handle_after_post_cook(post, cooked)
         next if restricted_category['value'] != category_id
       end
 
+      restricted_tags = automation.trigger_field('restricted_tags')
+      if tag_names = restricted_tags['value']
+        found = false
+        next if !post.topic
+
+        post.topic.tags.each do |tag|
+          found ||= tag_names.include?(tag.name)
```

Comment:
very minor, but I think we can break the loop when tag is found

Reply:
sure will add a break